packages/nimble-components/src/select/styles.ts

```diff
@@ -28,6 +40,81 @@ export const styles = css`
     [part='end'] {
         display: contents;
     }
+
+    .listbox {
+        overflow-x: clip;
+    }
+
+    .listbox.empty slot {
+        display: none;
+    }
+
+    .listbox.above {
+        flex-flow: column-reverse;
+    }
+
+    .search-field {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        height: ${controlHeight};
+        background: transparent;
+    }
+
+    .search-icon {
+        padding-left: ${smallPadding};
+    }
+
+    .search-field::after,
+    .search-field::before {
+        content: '';
+        position: absolute;
+        height: 0px;
+        border-bottom: rgba(${borderRgbPartialColor}, 0.15) 2px solid;
```

Comment:
Want to double check -- should this be out existing `dividerColor` token?

Reply:
No. To be clear, there is a `dividerBackgroundColor`, but not `dividerColor`. Additionally, this pattern is used elsewhere for similar visuals like the divider for the Combobox (which is what I copied directly). I'll note that in the designs it actually is spec'd to use 10% and not 15% transparency, so I've updated that part of the style.